### DISCUSSION TITLE
Update Grafana to the newest Version

### DISCUSSION THREAD

**Discussion by ElDiablo1985** [hassio-addons/addon-grafana#295] [General]
Hello, 

can you please Update the Grafana Home Assistant Addon to the newest Version ? 

Big Thanks. 

Best Regards,
ElDiablo

**Comment by frenck** [comment]
It will be updated once I do another maintenance cycle. There is not need to request it. It has no use.

../Frenck

**Reply by Matze5593** [reply]
but in the new versions it is possible to set other languages

**Reply by adandl** [reply]
I would also appreciate an update / not sure if that is true, but is grafana running at 8.3.0 (like the addon version indicates)?
thanks for the clarification
